packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -360,6 +361,7 @@ const selectInputProps = computed(() => {
                 :active="option.value === activeValue"
                 :icon="option.icon"
                 :density="props.density"
+                :truncation="props.truncation"
```

Comment:
shouldn't multiline also work with the same truncation?
![image](https://github.com/SchwarzIT/onyx/assets/151019977/2920499d-932d-4c59-94ca-c0c84cae4d13)